relayer/chain/substrate/chain.go

```diff
@@ -53,11 +61,13 @@ func (ch *Chain) SetReceiver(ethMessages <-chan []chain.Message, ethHeaders <-ch
 	return nil
 }
 
-func (ch *Chain) SetSender(subMessages chan<- []chain.Message, _ chan<- chain.Header) error {
+func (ch *Chain) SetSender(subMessages chan<- []chain.Message, _ chan<- chain.Header, beefyMessages chan<- store.DatabaseCmd) error {
 	listener := NewListener(
 		ch.config,
 		ch.conn,
```

Comment:
can rename to paraconn too

Reply:
See https://github.com/Snowfork/polkadot-ethereum/pull/314#discussion_r607245280